lib/src/main/scala/spinal/lib/bus/misc/BusSlaveFactory.scala

```diff
@@ -102,6 +102,12 @@ trait BusSlaveFactory extends Area{
   def readAddress(): UInt
   def writeAddress(): UInt
 
+  val readErrorFlag: Bool = False
+  val writeErrorFlag: Bool = False
```

Comment:
```suggestion
  val readErrorFlag: Bool = False.allowPruning
  val writeErrorFlag: Bool = False.allowPruning
```

I think it would be good to allow pruning, for protocols without errors, but I am not sure what pruning do, if it just removes warnings or if it removes unused signals from the generated RTL or something else.

EDIT: I've tried a few things and I think it just removes warnings, which is okay I guess?

Reply:
Done.